definitions/subworkflows/docm_cle.cwl

```diff
@@ -37,24 +36,43 @@ steps:
             docm_vcf: docm_vcf
             interval_list: interval_list
         out:
-            [docm_out]
+            [docm_raw_variants]
+    bgzip:
+        run: ../tools/bgzip.cwl
+        in:
+            file: GATK_haplotype_caller/docm_raw_variants
+        out:
+            [bgzipped_file]
+    index:
+        run: ../tools/index_vcf.cwl
+        in:
+            vcf: bgzip/bgzipped_file
+        out:
+            [indexed_vcf]
+    decompose:
+        run: ../tools/vt_decompose.cwl
+        in:
+            vcf: index/indexed_vcf
+        out:
+            [decomposed_vcf]
```

Comment:
In the detect_variants.cwl workflow an index step was required. Is one not required here?

Reply:
it's indexed again after the filter is applied, below.